spec/requests/lobbies_spec.rb

```diff
@@ -226,4 +226,64 @@
       expect(Pusher).to have_received(:trigger).with(lobby.code, Lobby::PLAYER_JOIN, { id: Player.last.id })
     end
   end
-end
+
+  describe 'start' do
+    let(:quiz) { FactoryBot.create(:quiz, user: user) }
+    let(:lobby) { FactoryBot.create(:lobby, quiz: quiz) }
+
+    before do
+      allow(Pusher).to receive(:trigger)
+    end
+
+    subject { post start_api_v1_lobby_path(id: lobby.id) }
+
+    it 'responds with successful HTTP status' do
+      subject
+
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'triggers a LOBBY_START Pusher event' do
+      subject
+
+      expect(Pusher).to have_received(:trigger).with(lobby.code, Lobby::LOBBY_START, {})
+    end
+  end
+
+  describe 'answer' do
```

Comment:
One more thing you could test would be the `context 'player already answered'` case and the API's response 😄